metadata.json

```diff
@@ -29,14 +29,12 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "6",
```

Comment:
to have a prettier changelog, we prefer one change in one PR. can you submit one that drops support for CentOS/RHEL/VZ 6 and one for Debian 8?

Reply:
ack ill split theses up later tonight